mne/viz/tests/test_evoked.py

```diff
@@ -78,181 +78,184 @@ def test_plot_evoked():
     import matplotlib.pyplot as plt
     rng = np.random.RandomState(0)
     evoked = _get_epochs().average()
```

Comment:
can you explain me the big diff in this file?

Reply:
There is no reason to have the `with warnings.catch_warnings` in there -- it could be masking useful warnings (and might have been, don't remember)